apps/vscode-extension/src/commands/extract/styling/helpers/utils.helper.ts

```diff
@@ -28,7 +28,7 @@ function generateScssVarName(selectors: string[], fileName: string): string {
   const wordsNotToReuseSet = new Set<string>([...forbiddenWords ?? [], '']);
   const prefixToAdd = workspace.getConfiguration('otter.extract.styling').get<string>('prefix');
 
-  selectors = [...([prefixToAdd ?? '']), fileName, ...selectors];
+  selectors = [prefixToAdd ?? '', fileName, ...selectors];
```

Comment:
with this syntax (which is the equivalent as before), I wonder if this is not a bug in fact :D

Reply:
Did we meant [...(prefixToAdd ? [prefixToAdd]: [])....] ?